c2corg_ui/static/js/suggestiondirective.js

```diff
@@ -0,0 +1,54 @@
+goog.provide('app.suggestionDirective');
+
+/**
+ * @param {angular.$compile} $compile Angular compile service.
+ * @param {angular.$sce} $sce Angular Strict Contextual Escaping service.
+ * @return {angular.Directive} Directive Definition Object.
+ * @ngInject
+ */
+app.suggestionDirective = function($compile, $sce) {
+  var html = '';
+  var header = '<span class="suggestion-text">' +
+                            '<span class="suggestion-title">' +
+                              '<span ng-bind-html="highlight(label, query)"></span>' +
+                              '<span ng-if="date_end">&nbsp;({{::date_end }})</span>' +
+                            '</span><br>' +
+                            '<span class="suggestion-info">' +
+                            '  <span ng-if="elevation">{{::elevation}}m</span>' +
+                            '</span>';
+  var footer =   '</span>';
```

Comment:
Honestly I would prefer having separated template functions even though it makes the code bigger. Else the code is going to be a big mess, really hard to understand, with conditions everywhere.
I think you should have a switch on ``doctype`` calling dedicated template functions, with a default template that can be used by default for document types with no special layout (with only the document title).